$NetBSD: patch-aa,v 1.3 2005/03/11 21:22:24 rillig Exp $

In NetBSD-1.6.2, the <stdlib.h> header uses the word bufsize as a
parameter name in a function prototype. The "file.h" header #defines
bufsize to 4096, which leads to a parser error.

--- src/cal.c.orig	Sun Mar  9 19:18:01 2003
+++ src/cal.c	Fri Mar 11 20:07:15 2005
@@ -19,14 +19,14 @@
  */
 
 /**************/
+#include <stdlib.h>
+#include <stdio.h>
+#include <time.h>
+
 #include "tree.h"
 #include "cli.h"
 #include "ui.h"
 #include "file.h"
-
-#include <stdlib.h>
-#include <stdio.h>
-#include <time.h>
 
 static char *const wday[] = 
 	{ "Sun", "Mon", "Tue",  "Wed", "Thu", "Fri", "Sat", "   "};
